app/src/components/HIAP/ClimateActionsSection.tsx

```diff
@@ -0,0 +1,144 @@
+            >
+              {t("citycatalyst-actions-title")}
+            </ButtonMedium>
+          </Box>
+          <Box
+            display="flex"
+            flexDirection="row"
+            gap="24px"
+            alignItems="center"
+          >
+            <Box display="flex" flexDirection="column" gap="8px">
+              <HeadlineSmall
+                fontFamily="heading"
+                fontWeight="bold"
+                color="content.secondary"
+              >
+                {t("top-actions-for-your-city")}
+              </HeadlineSmall>
+              <BodyLarge fontFamily="body" color="content.tertiary">
+                {t("top-actions-for-your-city-description")}
+              </BodyLarge>
+            </Box>
+            <Box>
+              <Button
+                bg="content.link"
+                color="white"
+                px="24px"
+                h="84px"
+                borderRadius="16px"
+                gap="12px"
+                onClick={onReprioritize}
+              >
+                <Icon
+                  as={LuRefreshCw}
+                  rotate={"270deg"}
+                  boxSize={"36px"}
+                  color="white"
+                />
+                <HeadlineSmall
+                  fontFamily="heading"
+                  fontWeight="bold"
+                  textTransform="none"
+                  color="white"
+                >
+                  {t("re-prioritize-actions")}
+                </HeadlineSmall>
+              </Button>
+            </Box>
+          </Box>
+        </Box>
+      ) : (
+        <Box display="flex" flexDirection="column" gap="24px" pb="24px">
+          <Box display="flex" flexDirection="column" gap="16px">
+            <TitleSmall
+              color="content.link"
+              fontFamily="heading"
+              fontSize="title.sm"
+              fontWeight="bold"
+              textTransform="uppercase"
+            >
+              {t("climate-actions-title")}
+            </TitleSmall>
+          </Box>
+          <Box
+            display="flex"
+            flexDirection="column"
+            gap="24px"
+            alignItems="center"
+          >
+            <Box display="flex" flexDirection="column" gap="8px">
+              <HeadlineSmall
+                fontFamily="heading"
+                fontWeight="bold"
+                color="content.secondary"
+              >
+                {t("top-actions-for-your-city")}
+              </HeadlineSmall>
+              <BodyLarge fontFamily="body" color="content.tertiary">
+                {t("actions-for-your-city-description")}
+              </BodyLarge>
+            </Box>
+            <Box
+              w="full"
+              display="flex"
+              flexDirection="row"
+              gap="8px"
+              alignItems="center"
+              border="1px solid"
+              borderColor="border.neutral"
+              borderRadius="8px"
+              p="16px"
+            >
```

Comment:
This might be a good use case for the Card component? Or are those styles too much for this?